bundles/org.openhab.binding.nuki/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -64,6 +90,27 @@
 		</config-description>
 	</thing-type>
 
+	<!-- Nuki Opener (Thing Type) -->
+	<thing-type id="opener" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Nuki Opener</label>
+		<description>Nuki Opener which is paired via Bluetooth to a Nuki Bridge.</description>
+		<channels>
+			<channel id="openerState" typeId="openerState"/>
+			<channel id="openerMode" typeId="openerMode"/>
+			<channel id="openerLowBattery" typeId="system.low-battery"/>
+			<channel id="ringActionState" typeId="ringActionState"/>
+			<channel id="ringActionTimestamp" typeId="ringActionTimestamp"/>
+		</channels>
+		<properties>
+			<property name="name">-</property>
+			<property name="firmwareVersion">-</property>
+		</properties>
+		<representation-property>nukiId</representation-property>
```

Comment:
You need a config parameter storing `nukiId`. I'm wondering how this thing is identified currently.

Reply:
The nukiId is stored as a property when the thing is created during discovery, I'll add it to properties.